flask_resources/__init__.py

```diff
@@ -157,10 +159,13 @@ def create_url_rules(self):
     request_body_parser,
     request_parser,
 )
+from .parsers.schema import WrapSchemaToPreserveContext
 from .resources import Resource, ResourceConfig, route
 from .responses import ResponseHandler, response_handler
 from .serializers import CSVSerializer, JSONSerializer, MarshmallowSerializer
 
+marshmallow.Schema = WrapSchemaToPreserveContext
```

Comment:
I'm not super fond of this since we're basically monkeypatching and thus rely on import order. Explicitly sub-classing from `WrapSchemaToPreserveContext` everywhere would be the best, but a much larger set of changes...

If we were to follow this, I would maybe go with the overall `invenio-base` approach, and thus make `marshmallow` a "global" dependency and utility.

Reply:
me neither. that is "only" the shorthand for not sub-classing every Schema by `WrapSchemaToPreserveContext` but if we want to use this solution rolling out the more explicit change will not be a big problem, it will only take some time